common/icons/index.js

```diff
@@ -1,13 +1,20 @@
 import arrow from './arrow';
-import clock from './clock';
+import {cc, cc_by, cc_nc, cc_nd, cc_pdm, cc_zero} from './licenses';
```

Comment:
I went with camelCase for zoomIn and zoomOut, but underscored is probably nicer? I'll update for consistency in another PR.

Reply:
Not sure, we use camelCase everywhere else?